frontera/settings/default_settings.py

```diff
@@ -50,6 +51,7 @@
 }
 SQLALCHEMYBACKEND_REVISIT_INTERVAL = timedelta(days=1)
 STATE_CACHE_SIZE = 1000000
+STATE_CACHE_SIZE_LIMIT = 0
```

Comment:
It is accessed here - https://github.com/scrapinghub/frontera/blob/master/frontera/contrib/backends/sqlalchemy/__init__.py#L41. Then it is passed to SQLACHEMY State object here - https://github.com/scrapinghub/frontera/blob/master/frontera/contrib/backends/sqlalchemy/components.py#L108. Then is passed to MemoryState via super and then the problem in the original comment arises.